doc/README.md

```diff
@@ -24,13 +24,17 @@ SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-excepti
 Documentation
 =============
 
-This directory contains documentation about various components in OpenJ9
+This directory contains the following types of documentation:
+
+* **`processes`**: Documentation about the processes used in OpenJ9.
```

Comment:
any reason not to put the categories in alphabetical order?

Reply:
If we want to have processes and release-notes first, followed by everything else in alphabetical order, perhaps there should be a division between the two categories of items.

Reply:
I was just tidying up to reflect what is there at the moment. I think there is more content that will end up here too. E.g. There's an action item in the community minutes to move the build instructions in here too. I avoided touching those for now as we need to coordinate/update the link at the website.

Maybe you can just let this through for now and we can come up with a better structure when the build instructions move. Seems we should separate the "task oriented" items from the "reference" items ... at least that makes sense to me.

Reply:
@pshipton Can this be merged please? The README is a lot more accurate than it was and we can deal with the order of items when I move the build instructions here.